Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ProjectController.cs

```diff
@@ -75,284 +76,169 @@ public class ProjectController : Controller
-                        foreach (var applicationUser in project.ApplicationUsers)
-                        {
-                            ProjectUser projectUser = new ProjectUser();
-                            projectUser.ProjectId = id;
-                            projectUser.UserId = applicationUser.Id;
-                            projectUser.CreatedBy = createdBy;
-                            projectUser.CreatedDateTime = DateTime.UtcNow;
-                            _projectRepository.AddUserProject(projectUser);
-                        }
-                        return Ok(project);
+                        ProjectUser projectUser = new ProjectUser();
+                        projectUser.ProjectId = projectId;
+                        projectUser.UserId = applicationUser.Id;
+                        projectUser.CreatedBy = createdBy;
+                        projectUser.CreatedDateTime = DateTime.UtcNow;
+                        await _projectRepository.AddUserProject(projectUser);
                     }
-                    else
-                    { return Ok(project); }
+                    return Ok(project);
                 }
-                return Ok(false);
-            }
-            catch (Exception ex)
-            {
-                ex.ToExceptionless().Submit();
-                throw ex;
+                else
+                { return Ok(project); }
             }
+            return Ok(false);
         }
 
         /**
-        * @api {put} api/Project/editProject 
+        * @api {put} api/project 
         * @apiVersion 1.0.0
-        * @apiName Project
+        * @apiName EditProject
         * @apiGroup Project
-        *  @apiParam {int} Id  Project Id
-        * @apiParam {string} Name  Project Name
-        * @apiParam {string} SlackChannelName  Project SlackChannelName
-        * @apiParam {bool} IsActive  Project IsActive
-        * @apiParam {int} TeamLeaderId  Project TeamLeaderId
-        * @apiParam {UserAc} ApplicationUsers  Project ApplicationUsers
+        * @apiParam {object} ProjectAc object 
         * @apiParamExample {json} Request-Example:
-        *      
-        *        {
-        *             "Id":"1",
-        *             "Name":"ProjectName",
-        *             "SlackChannelName":"SlackChannelName",
-        *             "IsActive":"True",
-        *             "TeamLeaderId":"1",
-        *             "ApplicationUsers":"List of Users"
-        *        }      
+        * {
+        *   "Id":"1",
+        *   "Name":"ProjectName",
+        *   "SlackChannelName":"SlackChannelName",
+        *   "IsActive":"True",
+        *   "TeamLeaderId":"1",
+        *   "ApplicationUsers":"List of Users"
+        * }      
         * @apiSuccessExample {json} Success-Response:
         * HTTP/1.1 200 OK 
         * {
-        *     "description":"edit Project in ProjectTable"
+        *     "description":"Update project information"
```

Comment:
As per you code when there is Ok response. It will return a Project json. Your comment does not reflect the actual output of the API